spec/xmltv.dtd

```diff
@@ -74,15 +74,15 @@
 <!ELEMENT icon EMPTY>
 <!ATTLIST icon src         CDATA #REQUIRED
                width       CDATA #IMPLIED
-               height      CDATA #IMPLIED> 
+               height      CDATA #IMPLIED>
 
 <!ELEMENT value (#PCDATA)>
 
 <!ELEMENT country (#PCDATA)>
 <!ATTLIST country lang CDATA #IMPLIED>
 
 <!ELEMENT episode-num (#PCDATA)>
-<!ATTLIST episode-num system CDATA "onscreen">
+<!ATTLIST episode-num system CDATA #REQUIRED>
```

Comment:
Where did you get this change? Or was it only a test because of your validation issues?

Original: https://github.com/XMLTV/xmltv/blob/master/xmltv.dtd

Reply:
Yeah i was testing here. I've never worked with DTD before, if you have it as "onscreen" does that mean it defaults to onscreen, or the attribute has to equal onscreen?

Reply:
Yes, as far as I can tell, it's a default value:
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms256140(v%3Dvs.100)